SPECS/libreswan.spec

```diff
@@ -20,66 +26,81 @@
     USE_LIBCURL=true \\\
     USE_LINUX_AUDIT=true \\\
     USE_NM=true \\\
+    USE_NSS_KDF=true \\\
     USE_SECCOMP=true \\\
-    USE_XAUTHPAM=true \\\
+    USE_AUTHPAM=true \\\
+    USE_DH2=true \\\
 %{nil}
 
-
 #global prever rc1
 
 Name: libreswan
 Summary: IPsec implementation with IKEv1 and IKEv2 keying protocols
 # version is generated in the release script
-Version: 3.26
-Release: %{?prever:0.}1.3%{?prever:.%{prever}}%{?dist}
+Version: 4.12
+Release: %{?prever:0.}2.1%{?prever:.%{prever}}%{?dist}.3
```

Comment:
the upstream seems to be 4.12-2.3 according to log message so we probably need something like 4.12-2.3.1 no?

Reply:
Done.